mmv1/products/accesscontextmanager/AuthorizedOrgsDesc.yaml

```diff
@@ -0,0 +1,130 @@
+    - !ruby/object:Api::Type::String
+      name: parent
+      input: true
+      required: true
+      description: |
+        Required. Resource name for the access policy which owns this `AuthorizedOrgsDesc`.
+      ignore_read: true
+    - !ruby/object:Api::Type::String
+      name: name
+      input: true
+      required: true
+      description: |
+        Resource name for the `AuthorizedOrgsDesc`. Format:
+        `accessPolicies/{access_policy}/authorizedOrgsDescs/{authorized_orgs_desc}`.
+        The `authorized_orgs_desc` component must begin with a letter, followed by
+        alphanumeric characters or `_`.
+        After you create an `AuthorizedOrgsDesc`, you cannot change its `name`.
+    - !ruby/object:Api::Type::Array
+      name: orgs
+      description: |
+        The list of organization ids in this AuthorizedOrgsDesc.
+        Format: `organizations/<org_number>`
+        Example: `organizations/123456`
+      item_type: Api::Type::String
+    - !ruby/object:Api::Type::Enum
+      name: assetType
+      input: true
+      description: |
+        The type of entities that need to use the authorization relationship during
+        evaluation, such as a device. Valid values are "ASSET_TYPE_DEVICE" and
+        "ASSET_TYPE_CREDENTIAL_STRENGTH".
+      values:
+        - :ASSET_TYPE_UNSPECIFIED
+        - :ASSET_TYPE_DEVICE
+        - :ASSET_TYPE_CREDENTIAL_STRENGTH
+    - !ruby/object:Api::Type::Enum
+      name: authorizationDirection
+      input: true
+      description: |
+        The direction of the authorization relationship between this organization
+        and the organizations listed in the "orgs" field. The valid values for this
+        field include the following:
+        
+        AUTHORIZATION_DIRECTION_FROM: Allows this organization to evaluate traffic
+        in the organizations listed in the `orgs` field.
+        
+        AUTHORIZATION_DIRECTION_TO: Allows the organizations listed in the `orgs`
+        field to evaluate the traffic in this organization.
+        
+        For the authorization relationship to take effect, all of the organizations
+        must authorize and specify the appropriate relationship direction. For
+        example, if organization A authorized organization B and C to evaluate its
+        traffic, by specifying "AUTHORIZATION_DIRECTION_TO" as the authorization
+        direction, organizations B and C must specify
+        "AUTHORIZATION_DIRECTION_FROM" as the authorization direction in their
+        "AuthorizedOrgsDesc" resource.
+      values:
+        - :AUTHORIZATION_DIRECTION_UNSPECIFIED
+        - :AUTHORIZATION_DIRECTION_TO
+        - :AUTHORIZATION_DIRECTION_FROM
+    - !ruby/object:Api::Type::Enum
+      name: authorizationType
+      input: true
+      description: |
+        A granular control type for authorization levels. Valid value is "AUTHORIZATION_TYPE_TRUST".
+      values:
+        - :AUTHORIZATION_TYPE_UNSPECIFIED
+        - :AUTHORIZATION_TYPE_TRUST
```

Comment:
Is there a server-side default values for these fields -- Does API return values for them if they are not explicitly specified?

Also should we remove `*_UNSPECIFIED` values? My understanding is that they are not valid values for users to specify.

Reply:
There are no server-side default values for `assetType`, `authorizationDirection`, or `authorizationType`. The user must specify them.

The `*_UNSPECIFIED` values exist under the hood, which is why I added them in the code here, but they cannot be used so IMO it's reasonable to remove them here.